language-examples/remedial-js/1.0-http-requests/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+    <title>Exercise 1.0 - HTTP Requests</title>
+    <script src="https://unpkg.com/systemjs@0.19.31/dist/system.src.js"></script>
+    <script src="https://code.angularjs.org/tools/typescript.js"></script>
+    <script src="system.config.js"></script>
+    <link href="https://unpkg.com/basscss@7.1.1/css/basscss.min.css" rel="stylesheet">
+</head>
+
+<!-- Display the application -->
+
+<body>
+    <p>HTTP Requests are composed of the following list of items in order:</p>
+    <ol>
+        <li>initial line, different for request vs. response</li>
```

Comment:
Its referring to "This line" so maybe change it to say:
"This being the first line of the request, depending on the type of request will be different for a GET request vs a POST request" and put an example of a first line like: "GET /api-endpoint-url HTTP/1.1"